export_scripts/export_metadata.py

```diff
@@ -0,0 +1,50 @@
+import oracledb
+import pandas as pd
+import api_keys
+
+query = """
+select 
```

Comment:
@kjallen I checked for duplicate ARKs in the Chron file, since we moved that to `ProjectItem.coverage` as a non-repeatable field:
```
$ tail -n +2 Chronological_period_covered.tsv | cut -f 1 | sort | uniq -d
21198/zz0009frh8
$ grep "21198/zz0009frh8" Chronological_period_covered.tsv
21198/zz0009frh8                1950-2000
21198/zz0009frh8                1950-2000
```
Just one repeated ARK, and the value is identical so no loss of data.  But it should be easy to filter out duplicate rows of all types via a `SELECT DISTINCT`, which I think would apply to all of the types, repeatable or not - there can be dup ARKs, but whole rows should all be unique.

Reply:
Confirming modification of production data:

Removed `NULL` entries for the following terms:

- Coverage (7)
- Title (2)
- Format (108)

Confirmed and removed duplicates

- Coverage (1)